### PR TITLE
Added support for TV episode summary, air date and rating. Added tests.

### PR DESCRIPTION
**Pull request by vkolova**

### CODE REVIEW

**Comment by anshulgoyal15** [comment]
would not work if the tv show is still going on and the scrapper job is to scrap data for each episode so use ```https://www.imdb.com/title/tt2193021/episodes?season=2``` **2** is the season and data returned must be an array with all the data for each episode

File: index.js [hunk 1]
```diff
@@ -26,6 +27,13 @@ function awardsPage(id) {
   }).catch(ifError)
 }
 
+function episodePage(id) {
```
Comment (anshulgoyal15) [inline index.js]: we need the summary , air time and rating for each episode.
method is suggested in comments

Comment (vkolova) [reply index.js]: Okay, working on it.

Comment (anshulgoyal15) [reply index.js]: great

**Pull request merged**